<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Designer</class>
 <widget class="QWidget" name="Designer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Designer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="DesignerView" name="designer_view" native="true"/>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0" colspan="2">
        <widget class="Line" name="lineUp">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="Line" name="lineDown">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="openButton">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QToolBox" name="toolBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>400</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_1">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>556</width>
            <height>307</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <attribute name="label">
           <string>Page 1</string>
          </attribute>
          <widget class="QGroupBox" name="groupBoxEdit">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>451</width>
             <height>171</height>
            </rect>
           </property>
           <property name="title">
            <string>Edit</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="buttonLine">
                <property name="text">
                 <string>&amp;Line</string>
                </property>
                <property name="shortcut">
                 <string>L</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="buttonPan">
                <property name="text">
                 <string>&amp;Pan</string>
                </property>
                <property name="shortcut">
                 <string>P</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="buttonErase">
                <property name="text">
                 <string>&amp;Erase</string>
                </property>
                <property name="shortcut">
                 <string>E</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="buttonFluid1">
                <property name="text">
                 <string>Fluid &amp;1</string>
                </property>
                <property name="shortcut">
                 <string>1</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="buttonFluid2">
                <property name="text">
                 <string>Fluid &amp;2</string>
                </property>
                <property name="shortcut">
                 <string>2</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="buttonBoundary">
                <property name="text">
                 <string>&amp;Boundary</string>
                </property>
                <property name="shortcut">
                 <string>B</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="buttonRectangle">
                <property name="text">
                 <string>B&amp;asin</string>
                </property>
                <property name="shortcut">
                 <string>R</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="page_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>100</width>
            <height>30</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Page 2</string>
          </attribute>
          <widget class="QGroupBox" name="groupBoxParameters">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>605</width>
             <height>341</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Parameters</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QGridLayout" name="gridLayoutParameters">
              <item row="6" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBoxHeight">
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <double>30000000.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
                <property name="value">
                 <double>300.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="16" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBoxDampingFactor">
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="17" column="0">
               <widget class="QLabel" name="labelShepard">
                <property name="text">
                 <string>Shepard</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="15" column="0">
               <widget class="QLabel" name="labelAlpha">
                <property name="text">
                 <string>Alpha</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="11" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBoxSamplingDistance">
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="minimum">
                 <double>0.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
                <property name="value">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="17" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBoxShepard">
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="11" column="0">
               <widget class="QLabel" name="labelSamplingDistance">
                <property name="text">
                 <string>Sampling distance</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="19" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBoxNoSlip">
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="18" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBoxXSPH">
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="19" column="0">
               <widget class="QLabel" name="labelNoSlip">
                <property name="text">
                 <string>No slip</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="13" column="1">
               <widget class="QSpinBox" name="spinBoxNeighbours">
                <property name="value">
                 <number>3</number>
                </property>
               </widget>
              </item>
              <item row="14" column="1">
               <widget class="QSpinBox" name="spinBoxC"/>
              </item>
              <item row="14" column="0">
               <widget class="QLabel" name="labelC">
                <property name="text">
                 <string>C</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="18" column="0">
               <widget class="QLabel" name="labelXSPH">
                <property name="text">
                 <string>Epsilon XSPH</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="13" column="0">
               <widget class="QLabel" name="labelNeighbours">
                <property name="text">
                 <string>Neighbours</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="15" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBoxAlpha"/>
              </item>
              <item row="16" column="0">
               <widget class="QLabel" name="labelDampingFactor">
                <property name="text">
                 <string>Damping factor</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="labelAccelerationX">
                <property name="text">
                 <string>Acceleration X</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="labelHeight">
                <property name="text">
                 <string>Scene height</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBoxAccelerationY">
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>9.810000000000000</double>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="labelWidth">
                <property name="text">
                 <string>Scene width</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBoxWidth">
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <double>300000.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
                <property name="value">
                 <double>300.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="9" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBoxAccelerationX"/>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="labelAccelerationY">
                <property name="text">
                 <string>Acceleration Y</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="11" column="2">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="page_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>100</width>
            <height>30</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Page 3</string>
          </attribute>
          <widget class="QWidget" name="gridLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>681</width>
             <height>291</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="action_save">
   <property name="text">
    <string>&amp;save</string>
   </property>
   <property name="toolTip">
    <string>Save scenario</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNewScene">
   <property name="text">
    <string>newScene</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionInc_size">
   <property name="text">
    <string>inc_size</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionDec_size">
   <property name="text">
    <string>dec_size</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionCheck_particles">
   <property name="text">
    <string>check_particles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>run</string>
   </property>
   <property name="toolTip">
    <string>Run simulation</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAddBoundary">
   <property name="text">
    <string>addBoundary</string>
   </property>
   <property name="toolTip">
    <string>Add boundary particles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionAddPhase1">
   <property name="text">
    <string>addPhase1</string>
   </property>
   <property name="toolTip">
    <string>Add phase 1 particles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionAddPhase2">
   <property name="text">
    <string>addPhase2</string>
   </property>
   <property name="toolTip">
    <string>Add phase 2 particles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>&amp;open</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DesignerView</class>
   <extends>QWidget</extends>
   <header>designerview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
